### ISSUE TITLE
Ember 3.1 deprecation warning

### ISSUE THREAD

**Issue by Rapid0o** [ember-decorators/argument#59]
I'm getting the following deprecation warning in Ember 3.1:

`DEPRECATION: 'propertyDidChange' is deprecated in favor of 'notifyPropertyChange'. It is safe to change this call to 'notifyPropertyChange'. [deprecation id: ember-metal.deprecate-propertyDidChange]`

[https://emberjs.com/deprecations/v3.x/#toc_use-notifypropertychange-instead-of-propertywillchange-and-propertydidchange](https://emberjs.com/deprecations/v3.x/#toc_use-notifypropertychange-instead-of-propertywillchange-and-propertydidchange)

On:
[https://github.com/ember-decorators/argument/blob/41b6dc983cca364b5154f7b880d9fd9c03f047f5/addon/-debug/utils/validation-decorator.js#L102](https://github.com/ember-decorators/argument/blob/41b6dc983cca364b5154f7b880d9fd9c03f047f5/addon/-debug/utils/validation-decorator.js#L102)
[https://github.com/ember-decorators/argument/blob/41b6dc983cca364b5154f7b880d9fd9c03f047f5/addon/-debug/utils/validation-decorator.js#L128](https://github.com/ember-decorators/argument/blob/41b6dc983cca364b5154f7b880d9fd9c03f047f5/addon/-debug/utils/validation-decorator.js#L128)

**Comment by rwjblue** [comment]
Yes, these will need to be updated. 

For addon authors this essentially means having to write:

Basically, with this:

```
const notifyPropertyChange = Ember.notifyPropertyChange || Ember.propertyDidChange;
```

Then use that `notifyPropertyChange` method. This will prevent the deprecation and continue to support older Ember versions (where `Ember.notifyPropertyChange` doesn’t exist).

**Comment by Rapid0o** [comment]
@rwjblue Ok, thnx for the explanation :+1:. I will create a PR.

**Comment by rwjblue** [comment]
This was fixed by #60
